fabfile.py

```diff
@@ -0,0 +1,180 @@
+import json
+import os
+import subprocess
+import sys
+from shlex import quote
+
+from invoke import run as local
+from invoke.context import Context
+from invoke.tasks import task
+
+LOCAL_DATABASE_NAME = "hypha"
+
+
+#############
+# Development
```

Comment:
Since we won't be deploying to Heroku I've removed all the tasks from this file except from the development one.